docker-images/pysyft-worker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.6-slim
+
+ENV WORKER_DIR /worker
+
+# Install PySyft
+RUN apt-get update && apt-get install -y gcc
+RUN pip install --no-cache-dir syft
+
+# Add worker code
+RUN mkdir $WORKER_DIR
```

Comment:
mkdir is redundant, WORKDIR creates the folder automatically :)

Reply:
thanks for the info. done